### PR TITLE
[android] Added some french translations

### PR DESCRIPTION
**Pull request by seckalou**

### CODE REVIEW

**Comment by mpimenov** [comment]
Hello. Please sign the CLA: https://maps.me/cla/

**Comment by seckalou** [comment]
> Hello. Please sign the CLA: https://maps.me/cla/

Hello,
Sorry I though I already did. Just signed it :)
Cheers

**Comment by mpimenov** [comment]
The correct file to fix is data/strings.txt.
Instructions are here: https://github.com/mapsme/omim/wiki/strings.txt

Also note that it is normal for some strings to be untranslated if they are part of an upcoming release. We usually bulk translate these strings closer to the release date, like in this PR: https://github.com/mapsme/omim/pull/11713

**Comment by seckalou** [comment]
Also, removed the translation covered in #11713. Should I generate the localisation after or before review?
Cheers

File: data/strings/strings.txt [hunk 1]
```diff
@@ -5478,6 +5479,7 @@
     en = General settings
     ru = Общие настройки
     fa = تنظیمات عمومی
+    fr = paramètres généraux
```
Comment (mpimenov) [inline data/strings/strings.txt]: s/paramètres/Paramètres/

**Comment by mpimenov** [comment]
We usually go with after.
Please collapse your two commits into one and then generate localizations in a separate commit.

File: data/strings/strings.txt (hunk 1 above)
Comment (seckalou) [reply data/strings/strings.txt]: Done!

**Pull request merged**